docs/install.rst

```diff
@@ -55,7 +56,26 @@ If you'd like to serve the public tenant for unrecognised hostnames instead, use
         #...
     )
 
-Make sure you have ``django.template.context_processors.request`` (``django.core.context_processors.request`` if you're on Django 1.8) listed under ``TEMPLATE_CONTEXT_PROCESSORS`` else the tenant will not be available on ``request``.
+Make sure you have ``django.template.context_processors.request`` listed under ``context_processors`` (Django 1.8+):
```

Comment:
Django versions older the 1.8 are unsupported - remove special case comments and any examples for unsupported versions along with it.